modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1592,85 +1591,70 @@ private void sendRecordingUpdate(Recording recording) {
   }
 
   @Override
-  public void repopulate(final String indexName) {
+  public void repopulate(final String indexName) throws SchedulerServiceDatabaseException {
     notEmpty(indexName, "indexName");
 
-    final String destinationId = SchedulerItem.SCHEDULER_QUEUE_PREFIX + WordUtils.capitalize(indexName);
-    final Organization organization = new DefaultOrganization();
-    final User user = SecurityUtil.createSystemUser(systemUserName, organization);
-    SecurityUtil.runAs(securityService, organization, user, () -> {
-      int current = 0;
+    final String destinationId = SchedulerItem.SCHEDULER_QUEUE_PREFIX + indexName.substring(0, 1).toUpperCase()
```

Comment:
Doesn't substring() return characters?  What if we end up with multiple indexes with the same first letter under the scheduler queue prefix?

Reply:
`.substring(1)` returns the String except for the first letter. In other words, the input here is something like `adminui` and the required output we need is `Adminui`. This is just copied from the workflow index rebuild btb ;-)